waik/waik_finish.erb

```diff
@@ -0,0 +1,98 @@
+<%#
+#kind: finish
+#name: WAIK default
+#oses:
+#- Windows 6.1
+#- Windows 6.3
+# Parameters are expected to be set in Foreman (globally or per group/host)
+params:
+- windowsLicenseKey: ABCDE-ABCDE-ABCDE-ABCDE-ABCDE # Valid Windows license key
+- windowsLicenseOwner: Company, INC # Legal owner of the Windows license key
+- localAdminiAccountDisabled: false
+- ntpSever: time.windows.com,other.time.server
+- domainAdminAccount: administrator@domain.com # use this account to join domain
+- domainAdminAccountPasswd: Password for the domain Admin account
```

Comment:
The user to join the domain doesn't need to be admin.
Also, other templates don't use camelCase for params.